### DISCUSSION TITLE
modify fingerprints

### DISCUSSION THREAD

**Discussion by DeltaDarkness** [kaliiiiiiiiii/Selenium-Driverless#105] [Q&A] [answered]
can i modifyfingerprints as undetects apps ?

**Comment by ttlns** [comment, marked as answer]
No, this is as now not supported. Please only open an issue when there's an actual bug!

**Reply by DeltaDarkness** [reply]
Ok thanks
and i am sorry

**Reply by ttlns** [reply]
No worries:)
